<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultxss</class>
 <widget class="QMainWindow" name="resultxss">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Result</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xss.qrc">
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="India"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="outputxss">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 0);</string>
        </property>
        <property name="locale">
         <locale language="English" country="India"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>URL</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>HTTP-Code</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Result</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>158</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(0, 0, 0);
font: 75 14pt &quot;Bitstream Vera Sans Mono&quot;;</string>
        </property>
        <property name="text">
         <string>Possible XSS Injections</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>158</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="outputsuccessfull_vectors">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 0);</string>
        </property>
        <property name="locale">
         <locale language="English" country="India"/>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>URL</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
font: 75 12pt &quot;Bitstream Vera Sans Mono&quot;;
color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string> Total Attack Vectors   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="total_vectors">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;Bitstream Vera Sans Mono&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
font: 75 12pt &quot;Bitstream Vera Sans Mono&quot;;
color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>      Total Time  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="total_time">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;Bitstream Vera Sans Mono&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
font: 75 12pt &quot;Bitstream Vera Sans Mono&quot;;
color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string> Successful Vectors     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="successfull_vectors">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;Bitstream Vera Sans Mono&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
font: 75 12pt &quot;Bitstream Vera Sans Mono&quot;;
color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string> Failed Vectors   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="failed_vectors">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;Bitstream Vera Sans Mono&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QProgressBar" name="resultprogressbar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="start_button">
        <property name="styleSheet">
         <string notr="true">
font: 14pt &quot;Abyssinica SIL&quot;;
</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="quit_button">
        <property name="styleSheet">
         <string notr="true">
font: 14pt &quot;Abyssinica SIL&quot;;
</string>
        </property>
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="BrowserButton">
        <property name="styleSheet">
         <string notr="true">
font: 14pt &quot;Abyssinica SIL&quot;;
</string>
        </property>
        <property name="text">
         <string>Open Results In Browser</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="xss.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quit_button</sender>
   <signal>pressed()</signal>
   <receiver>resultxss</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
